docs/contribute/documentation.md

```diff
@@ -14,6 +14,44 @@ please follow these guidelines:
 * Keep the documentation **short and clear** (e.g. by using lists)
 * Avoid new nested link structures and link lists
 
+## Write IDE specific documentation
+Use the following tags if you want to embed an IDE specific part into a documentation or want to provide IDE specific versions of a page.
+
+**Markdown:**
+```markdown
+{{ "{% capture eclipse " }}%}
+## Eclipse content here
+Eclipse content here
+{{ "{% endcapture " }}%}
+{{ "{% capture intellij " }}%}
+## Intellij content here
+Intellij content here
+{{ "{% endcapture " }}%}
+{{ "{% include ide-tabs.html eclipse=eclipse intellij=intellij " }}%}
+```
+**Rendered Markdown:**
+{% capture eclipse %}
+## Eclipse content here
+Eclipse content here
+{% endcapture %}
+{% capture intellij %}
+## Intellij content here
+Intellij content here
+{% endcapture %}
+{% include ide-tabs.html eclipse=eclipse intellij=intellij %}
+
+
+**Attention:** The major disadvantage of this solution is that we use [jekyll liquid tags](https://jekyllrb.com/docs/liquid/).
+Therefore the content of the documentation could break the documentation build! This happens if you have typo in your tags or forgot something etc.
+If you want to make sure that the documentation works, compile the documentation (see below for more information).
+
+## Reference IDE specific documentation
+
+You can reference the IDE specific version of a page via an url parameter `tab=<ide>` (e.g. `<url>?tab=intellij` or `<url>?tab=eclipse`).
```

Comment:
This works in this example, but, as I have already noted in my other comments, does not specify whether to use the `.md` or `.html` ending for the references.
Normally, GitHub pages would use the `.md` ending everywhere as it allows you to also navigate the references in the source view and they are automatically resolved when building the pages. But, this resolution does not work with tab-specific links. The question is now whether there is some workaround for this (or another logic to replace the current tab specifier in the URL) or whether we want to directly use the `.html` reference for such references. This should then also be mentioned in this section.